libcaf_core/caf/local_actor.hpp

```diff
@@ -77,6 +77,27 @@ class CAF_CORE_EXPORT local_actor : public monitorable_actor {
     telemetry::int_gauge* mailbox_size = nullptr;
   };
 
+  /// Optional metrics for inbound stream traffic collected by individual actors
+  /// when configured to do so.
+  struct inbound_stream_metrics_t {
+    /// Counts the total number of processed stream elements from upstream.
+    telemetry::int_counter* processed_elements = nullptr;
+
+    /// Tracks how many stream elements from upstream are currently buffered.
+    telemetry::int_gauge* input_buffer_size = nullptr;
```

Comment:
I think we should keep it. The word "buffer" ist just too generic otherwise. One would think that a buffer of an inbound path probably caches inputs, but calling input-buffer makes it obvious.